.github/workflows/build.yml

```
@@ -10,15 +10,15 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v6
         with:
           fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
       - name: Set up JDK 17
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v5
```

actionlint (1.7.11)

[error] missing input "distribution" which is required by action "actions/setup-java@v5". all required inputs are "distribution"

(action)